testsuite/integration/basic/pom.xml

```diff
@@ -224,6 +227,9 @@
                                         <exclude>org/jboss/as/test/integration/transaction/inflow/TransactionInflowTestCase.java</exclude>
                                         <exclude>org/jboss/as/test/integration/jsr77/**/*TestCase*.java</exclude>
                                     </excludes>
+                                    <includes>
+                                        <include>org/jboss/as/test/integration/web/security/authentication/*TestCase.java</include>
```

Comment:
This means that you are only going to run your test, which excludes over 1000 existing tests